cloudwatchlogs/cloudwatchlogs_lambda.js

```diff
@@ -19,7 +19,7 @@ var sourceNameOverride = process.env.SOURCE_NAME_OVERRIDE || 'none';          //
 var encoding = process.env.ENCODING || 'utf-8';  // default is utf-8
 
 // Include logStream and logGroup as json fields within the message. Required for SumoLogic AWS Lambda App
-var includeLogInfo = false;  // default is true
+var includeLogInfo = false;  // default is false
```

Comment:
I noticed that the "Use a blueprint" version in the AWS console still has `true` here. Is that intentional?

![image](https://user-images.githubusercontent.com/23050/61951816-5f808680-af6f-11e9-8b60-02e3321463f9.png)

![image](https://user-images.githubusercontent.com/23050/61951861-7d4deb80-af6f-11e9-900d-e0cf4be3383a.png)

Reply:
the blueprint uses old version so that's why it's true there